src/eager/readonly_tensor.rs

```diff
@@ -32,6 +36,38 @@ impl<T: TensorType> Deref for ReadonlyTensor<T> {
     }
 }
 
+impl<T: TensorType> Debug for ReadonlyTensor<T> {
```

Comment:
ReadonlyTensor could use a Display impl, too, to keep it consistent with Tensor.